### PR TITLE
redesign complete

### PR DESCRIPTION
**Pull request by scottsgcho**

### CODE REVIEW

File: components/profile/liquidity.js [hunk 1]
```diff
@@ -31,27 +48,162 @@ const Liquidity = props => {
...
+  async function fetchMemberUsdcHistory(skip) {
+    let newHistory = await getMemberUsdcHistory({
+      params: {
+        ethAddress,
+        skip: skip ? skip : 0
+      },
+      store
+    })
+    console.log(usdcHistory)
+    let oldHistory = usdcHistory ? usdcHistory : []
```
Comment (1kevinxu1) [inline components/profile/liquidity.js:60]: Can we make a comment here that this is for pagination?

File: components/profile/liquidity.js [hunk 2]
```diff
@@ -31,27 +48,162 @@ const Liquidity = props => {
...
+
+  async function fetchMemberDntHistory(skip) {
+    let newHistory = await getMemberDntHistory({
+      params: {
+        ethAddress,
+        skip: skip ? skip : 0
+      },
+      store
+    })
+    let oldHistory = dntHistory ? dntHistory : []
```
Comment (1kevinxu1) [inline components/profile/liquidity.js:73]: Same with here... I feel like the "old history" + "new history" variables aren't intuitive when reading the code (makes sense when I read the logic tho), no need to worry about the variables but I think adding context here will help for future iterations

File: pages/network.js [hunk 3]
```diff
@@ -15,10 +15,10 @@ import { format } from '../utils/money'
 
 const Network = props => {
   const store = useStoreApi()
-  const { getProtocol, setShowProfile } = store
+  const { getProtocol, setShowProfile, getMember } = store
 
   const [epochSelectorOpen, setEpochSelectorOpen] = useState(false)
-  const [selectedEpoch, setSelectedEpoch] = useState(getProtocol().epochNumber)
+  const [selectedEpoch, setSelectedEpoch] = useState(getProtocol().epochNumber - 1)
```
Comment (1kevinxu1) [inline pages/network.js:21]: switching to epochNumber -1 because we only show historical state.

File: pages/pool.js [hunk 4]
```diff
@@ -23,12 +34,16 @@ const Pool = props => {
...
   useEffect(() => {
-    getPoolInformation()
+    fetchPool()
+    fetchMemberUsdcHistory()
+    fetchMemberDntHistory()
   }, [])
 
-  async function getPoolInformation() {
+  async function fetchPool() {
     let pool = await getPool({
```
Comment (1kevinxu1) [inline pages/pool.js:47]: Isn't there already a fetchPool/getPool call pattern in the dnt.js? Why are we calling it again for this page?

File: pages/pool.js [hunk 5]
```diff
@@ -43,72 +58,195 @@ const Pool = props => {
     setMemberPool(memberPool)
   }
 
+  async function fetchMemberUsdcHistory(skip) {
```
Comment (1kevinxu1) [inline pages/pool.js:61]: Similar to the getPool/fetchPool, why is there a similar call pattern for this USDC/DNT history in pool.js?

**Comment by 1kevinxu1** [comment]
Overall this looks fine, most of the line changes are css/styling changes so nothing major and can probably be caught when we look at it on the app.

My only concern which is kind of major is the repeat back-end calls across the different pages—getPool, fetchPool, etc. there seems to be a lot of re-used code. Something doesn't seem right with the way we are managing state if we are having a lot of the same logic across our components. Let's discuss in the discord channel.

File: api/get.js [hunk 6]
```diff
@@ -234,4 +237,71 @@ export const getNetwork = async ({params, store}) => {
...
+    console.log("getStakeRanking: ", err)
+    if (!store) return
+    store.setShowToast({ show: true, text: "Couldn't get stake rankings. Please try again later", reason: 'error' })
+  }
+}
+
+export const getMemberStakeHistory = async ({params, store}) => {
+  try {
+    //params: ethAddress, page, epoch
+    let { data: { result } } = await axios.get(process.env.API_URL + '/api/ctPools/dnt/stakeHistory', {
```
Comment (zaqk) [inline api/get.js:294]: is this supposed to be let?

**Comment by zaqk** [comment]
done reviewing. looks fine to me nothing pops out.

**Pull request merged**